app/assets/stylesheets/homepage.scss

```diff
@@ -0,0 +1,39 @@
+$coral: #ec4339;
+$white: #ffffff;
+
+.title-box {
+  background-color: $coral;
+  font-family: 'Alegreya Sans SC', sans-serif;
```

Comment:
Properties should be ordered background-color, color, font-family, font-size, font-size, font-weight, letter-spacing, padding